docs: tw-1012-finality (#3537)
OffchainLabs/arbitrum-docs: docs/how-arbitrum-works/deep-dives/sequencer.mdx

  - [Batching](#batching-and-compression)
  - [Compression](#compression)
  - [Submitting to the Sequencer Inbox contract](#submitting-to-the-sequencer-inbox-contract)

> - [Finality](#finality): A summary of the soft and hard finality levels, with a pointer to the full [Finality](/how-arbitrum-works/deep-dives/finality.mdx) page (not a Sequencer task).

Reviewer: ```suggestion
- [Finality](#finality): A summary of the soft and <a data-quicklook-from="hard-finality">hard finality</a> levels, with a pointer to the full [Finality](/how-arbitrum-works/deep-dives/finality.mdx) page (not a Sequencer task).
```

- **Soft-confirmation receipts**: The Sequencer’s unique ability to immediately process and include transactions allows it to provide users with instant, “soft-confirmation” receipts
- **Parent chain finality**: Once batches post, the transactions achieve parent-chain-level finality, secured by the parent chain’s consensus mechanism.

By efficiently sending compressed transaction batches to the Sequencer Inbox contract using the most cost-effective method available, the Sequencer ensures transactions are securely recorded on the parent chain. This process maintains the integrity and reliability of the network, enabling users to perform fast, secure transactions.

## Finality

Arbitrum offers two levels of finality:

1. **Soft finality**: The Sequencer’s real-time feed confirms transactions almost instantly, but those confirmations are provisional. They rely on the Sequencer sequencing honestly and staying available.

> 2. **Hard finality**: Transactions are included in batches posted to and finalized on the parent chain, inheriting the parent chain’s security. Final settlement happens when the Rollup protocol confirms the containing [assertion](/how-arbitrum-works/deep-dives/assertions.mdx), after the [BoLD](/how-arbitrum-works/bold/gentle-introduction.mdx) challenge period.

Reviewer: ```suggestion
2. **Hard finality**: The Sequencer's batches are posted to and finalized on the parent chain, and the batch's transactions inherit the parent chain's security. Final settlement happens when the Rollup protocol confirms the containing [assertion](/how-arbitrum-works/deep-dives/assertions.mdx), after the [BoLD](/how-arbitrum-works/bold/gentle-introduction.mdx) challenge period.
```

> Posting a batch is not itself final settlement. The transaction data becomes available and inherits the parent chain’s data-layer security, but the resulting child chain state settles only once an assertion is confirmed.

Reviewer: ```suggestion
Posting a batch is not itself final settlement. The transaction data becomes available and inherits the parent chain’s data-layer security, but the resulting child <a data-quicklook-from="chain-state">chain state</a> settles only once an assertion is confirmed.
```
